packages/components/src/components/tabs/tabs.jsx

```diff
@@ -119,47 +129,56 @@ const Tabs = ({
                         'dc-tabs__list--center': center,
                         'dc-tabs__list--header-fit-content': header_fit_content,
                         [`dc-tabs__list--${className}`]: className,
+                        'dc-tabs__list--overflow-hidden': is_scrollable,
                     })}
                     ref={tabs_wrapper_ref}
                 >
-                    {React.Children.map(children, (child, index) => {
-                        if (!child) return null;
-                        const { count, header_content, icon, label, id } = child.props;
-                        return (
-                            <Tab
-                                active_icon_color={active_icon_color}
-                                className={className}
-                                count={count}
-                                icon={icon}
-                                icon_color={icon_color}
-                                icon_size={icon_size}
-                                is_active={index === active_tab_index}
-                                key={label}
-                                is_label_hidden={children.length === 1 && single_tab_has_no_label}
-                                label={label}
-                                id={id}
-                                top={top}
-                                bottom={bottom}
-                                header_fit_content={header_fit_content}
-                                active_tab_ref={index === active_tab_index ? active_tab_ref : null}
-                                header_content={header_content}
-                                onClick={() => onClickTabItem(index)}
-                                setActiveLineStyle={setActiveLineStyle}
+                    <ThemedScrollbars
+                        className='dc-themed-scrollbars-wrapper'
+                        is_only_horizontal={true}
```

Comment:
Do we need an `is_only_horizontal` prop here since we don't need/want a scroll in the desktop version? 
IIRC there's a `has_horizontal` prop which we can use when it's mobile.

Reply:
We do as we get vertical scroll instead, and there is no horizontal scroll on the desktop version :)

Reply:
What about `has_horizontal` for Mobile?

Reply:
we don't need it here, it works without this prop